nautobot/circuits/homepage.py

```diff
@@ -0,0 +1,34 @@
+from nautobot.circuits.models import Circuit, Provider
+from nautobot.core.apps import HomePageColumn, HomePageItem, HomePagePanel
+
+
+layout = (
+    HomePageColumn(
```

Comment:
I don't particularly like apps having to pick which column to place their panel into. In my utopian vision, we would just define a flat list (ordered by weight of course) of all HomePagePanels across the entire application, and the page rendering code itself would automatically partition them into approximately equal-length columns as appropriate. That way, if for example we have a whole bunch of plugins that are all adding panels to the end of the list (high weight), we don't end up with an overly-long third column while columns 1 and 2 remain short.